etna/records/fields.py

```diff
@@ -18,12 +18,13 @@ class LazyRecord(SimpleLazyObject):
     For example, when requesting a ``Record`` attribute value other than
     ``iaid``, or requesting a string representation of it.
     """
+    records_client = get_records_client()
 
     def __init__(self, iaid: str):
         self.__dict__["iaid"] = iaid
 
         def _fetch_record():
-            return Record.api.fetch(iaid=iaid)
+            return self.records_client.fetch(iaid=iaid)
```

Comment:
It would be great to get your thoughts on these changes @ababic. Currently when loading the Record model no longer has the `api` method. I'm not 100% clear on the changes to this model underneath so any clarification would be appreciated!